code/src/components/Artist.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import './artist_style.css'
+
+export const ArtistArray = (props) => {
+  // console.log(props)
+  return  (
+    <a className="artist-name" href="http://" > {props.name} </a>
```

Comment:
If you don't have anything in the links href the standard is to set `#` there
```suggestion
    <a className="artist-name" href="#" > {props.name} </a>
```